Client/Frontend/Browser/BraveSkusWebHelper.swift

```diff
@@ -82,6 +82,50 @@ class BraveSkusWebHelper {
       return nil
     }
   }
+  
+  var environment: String? {
+    guard let host = url.host else { return nil }
+    return Self.environment(domain: host)
+  }
+  
+  static func environment(domain: String) -> String? {
+    switch domain {
+    case "account.brave.software", "vpn.brave.software": return "development"
+    case "account.bravesoftware.com", "vpn.bravesoftware.com": return "staging"
+    case "account.brave.com", "vpn.brave.com": return "production"
+    default:
+      return nil
+    }
```

Comment:
convert environment string to a string enum? its very easy to make mistakes typing strings. at least for me lol